From: Rene Kita <mail@rkta.de>
Date: Thu Sep 26 19:05:27 CEST 2024
Subject: Remove islang.c
Patch: patches/Remove-islang.c.patch

Seems to be unused. VCS history does not show anything useful, mostly
untouched since 2001.

---
 islang.c |   41 -----------------------------------------
 1 file changed, 41 deletions(-)

--- a/islang.c
+++ /dev/null
@@ -1,41 +0,0 @@
-/* vi: set sw=4 ts=8 ai sm noet : */
-/*
- * Copyright (c) 2000, NBG01720@nifty.ne.jp
- *
- * To compile this program:
- *      gcc -Zomf -Zcrtdll -O2 -Wall -s islang.c
- */
-#define INCL_DOSNLS
-#include <os2.h>
-#include <stdlib.h>
-#include <stdio.h>
-#include <string.h>
-#include <ctype.h>
-
-int
-main(int argc, char **argv)
-{
-    if (argc <= 1)
-	return 1;
-
-    if (isdigit((int)*argv[1])) {
-	unsigned long CpList[8], CpSize;
-	APIRET rc = DosQueryCp(sizeof(CpList), CpList, &CpSize);
-	if (rc)
-	    return rc;
-	while (--argc > 0)
-	    if (*CpList == atoi(argv[argc]))
-		return 0;
-    }
-    else {
-	char *lang = getenv("LANG");
-	if (!lang || !*lang) {
-	    lang = getenv("LANGUAGE");
-	    if (!lang || !*lang)
-		return 1;
-	}
-	if (!strnicmp(lang, argv[1], 2))
-	    return 0;
-    }
-    return 1;
-}
